AUTHORS

```diff
@@ -3490,5 +3490,17 @@ John Gee <john@ruru.gen.nz>
 Hrishikesh Kadam <hrkadam.92@gmail.com>
 KrayzeeKev <github@lentinfamily.com>
 Airing <airing@ursb.me>
+jiahao.si <sijiahao0839@gmail.com>
+Rhys <Anemy@users.noreply.github.com>
+Keyhan Vakil <kvakil@github.kvakil.me>
+Mark S. Miller <erights@users.noreply.github.com>
+Jianru Lin <jianru.lin@gmail.com>
+Lenvin Gonsalves <41874033+98lenvi@users.noreply.github.com>
+Andreu Botella <andreu@andreubotella.com>
+Keyhan Vakil <kvakil@sylph.kvakil.me>
```

Comment:
This is a duplicate that will need a `.mailmap` entry. Otherwise, this all LGTM>